src/networking/responses/products-response.ts

```diff
@@ -1,3 +1,23 @@
+export interface Price {
+  amount: number;
+  currency: string;
+}
+
+export interface PurchaseOptionPrice {
+  period_duration: string | null;
+  price: Price | null;
```

Comment:
Hmm in the end, I think the `price` should never be null? And I'm not sure about the `period_duration`....

Reply:
The price is null for trials

Reply:
Ahh right makes sense. Same question about the duration though

Reply:
In case of the period_duration that was supposed to be reused for one-offs.

Reply:
Makes sense, thanks!